try_test.go

```diff
@@ -141,8 +141,9 @@ func TestFrame(t *testing.T) {
 func TestF(t *testing.T) {
 	buf := new(strings.Builder)
 	logger := log.New(buf, "", 0)
+	logger.SetFlags(log.Lshortfile)
```

Comment:
The `0` in the line above is flags, so you can change it to `log.Lshortfile` instead of calling `SetFlags`.

Reply:
Done.